### PR TITLE
[UI] Assets

### LINKED ISSUE #2: [UI] Assets
### Description:
We need to download all required assets, including images and icons, and organize them within our project. Icons should be downloaded in .svg format. Each asset should be properly named, and classes for AppImages and AppIcons should be created for easy reference. Maintain the following folder structure:

```
assets
  ├── images (contains all images)
  ├── icons (contains all icons)

```
### Tasks:
**Download Assets:**

- Identify and download all necessary images and icons.
- Ensure all icons are in .svg format.

**Organize Folder Structure:**

- Create an assets folder in the root directory.
- Within assets, create images and icons subfolders.
- Place images in the images folder and icons in the icons folder.

**Naming Convention:**

- Ensure all asset files have descriptive and consistent names.

**Create Asset Classes:**

- Create a Dart file app_images.dart for managing image assets.
- Create a Dart file app_icons.dart for managing icon assets.
- Define constants in these files for referencing the assets.

**Acceptance Criteria:**

- All assets are downloaded and organized into the appropriate folders.
- Asset files have clear, descriptive names.
- app_images.dart and app_icons.dart files are created with constants referencing the assets.

### Example Implementation:
**app_images.dart**

```
class AppImages {
  static const String _baseImageUrl = 'assets/images';

  static const String logo = '$_baseImageUrl/logo.png';
  static const String background = '$_baseImageUrl/background.jpg';
  // Add more image references as needed
}

```
**app_icons.dart**

```
class AppIcons {
  static const String _baseIconUrl = 'assets/icons';

  static const String home = '$_baseIconUrl/home.svg';
  static const String settings = '$_baseIconUrl/settings.svg';
  // Add more icon references as needed
}

```

### PR DESCRIPTION
**Pull request by CodeWithSuleman**
Add AppImages and AppIcons for consistent UI Assets

### CODE REVIEW

**Comment by AbdullahAhmedSoomro** [comment]
PR Name:
[UI] Assets
And PR dev m merge hogi ap ne master m request ki hai

**Comment by AbdullahAhmedSoomro** [comment]
Commit Message:
- Added Images and Icons.

File: assets/icons/Group 1000000743.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/Group 1000000743.svg:1]: No need for this image, we just need that icon without background color

Comment (AbdullahAhmedSoomro) [reply assets/icons/Group 1000000743.svg:1]: And i think flutter has this icon, so delete this one

File: assets/icons/Time.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/Time.svg:1]: flutter has this icon, so delete this one

File: assets/icons/add.png
Comment (AbdullahAhmedSoomro) [inline assets/icons/add.png:1]: And i think flutter has this icon, so delete this one

File: assets/icons/back_arrow.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/back_arrow.svg:1]: And i think flutter has this icon, so delete this one

File: assets/icons/cart.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/cart.svg:1]: Check this one we have this one too in flutter may be, if we don't, we just need the icon not the background color

File: assets/icons/delete.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/delete.svg:1]: flutter has this icon, so delete this one

File: assets/icons/down_arrow.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/down_arrow.svg:1]: flutter has this icon, so delete this one

File: assets/icons/heart.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/heart.svg:1]: flutter has this icon, so delete this one

File: assets/icons/home.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/home.svg:1]: flutter has this icon, so delete this one

File: assets/icons/mic.svg
Comment (AbdullahAhmedSoomro) [inline assets/icons/mic.svg:1]: flutter has this icon, so delete this one

**Review by AbdullahAhmedSoomro — CHANGES_REQUESTED** [review]
Changes requested

File: assets/icons/Group 1000000743.svg
Comment (CodeWithSuleman) [reply assets/icons/Group 1000000743.svg:1]: ok

**Review by AbdullahAhmedSoomro — APPROVED** [review]

**Pull request merged**